tests/setup

```diff
@@ -12,6 +12,9 @@ JQTESTDIR=$(cd "$(dirname "$0")" && pwd)
 JQBASEDIR=$JQTESTDIR/..
 JQ=${JQ:-$JQBASEDIR/jq}
 
+LC_ALL=C
+export LC_ALL
+
```

Comment:
Maybe a comment about making locale constant because of locale dependent output?

Reply:
Good idea